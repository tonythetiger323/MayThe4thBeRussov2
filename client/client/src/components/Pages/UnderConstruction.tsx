import * as React from 'react'
import Page from '../Layouts/Page'
import Typography from '@material-ui/core/Typography'

const UnderConstruction = (): JSX.Element => {
  return (
    <div>
      <Page>
        <Typography variant='h1'>Under Construction</Typography>
      </Page>
    </div>
  )
}

export default UnderConstruction
