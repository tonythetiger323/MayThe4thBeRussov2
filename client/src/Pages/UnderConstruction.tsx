import * as React from 'react';
import Typography from '@material-ui/core/Typography';
import Grid from '@material-ui/core/Grid'
import Card from '@material-ui/core/Card';
import CardMedia from '@material-ui/core/CardMedia';
import CardContent from '@material-ui/core/CardContent';
import { withStyles } from '@material-ui/core/styles';

const styles = {
  card: {
    maxWidth: 650,
  },
  media: {
    height: 352,
  },
};

const UnderConstruction = (props: any): JSX.Element => {
  const { classes } = props;
  return (
  <div>
    <Grid container direction='column' alignContent='center' alignItems='center' justify='center'>
      <Typography variant='h1'>Under Construction</Typography>
      <Card className={classes.card}>
        <CardMedia className={classes.media}
        image='https://res.cloudinary.com/tonythetiger323/image/upload/v1551236759/MayThe4thBeRusso/Other%20Photos/Together/FullSizeRender.jpg'
        title='Megan and Randy'
        />
        <CardContent>
          <Typography variant='h3'>This page is currenlty under construction, please check back soon!</Typography>
        </CardContent>
      </Card>
    </Grid>
  </div>
  )

}

export default withStyles(styles)(UnderConstruction)
