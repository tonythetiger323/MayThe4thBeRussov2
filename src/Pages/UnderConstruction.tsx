import * as React from 'react'
import Typography from '@material-ui/core/Typography'

const UnderConstruction = (): JSX.Element => {
  return <Typography variant='h1'>Under Construction</Typography>
}

export default UnderConstruction
