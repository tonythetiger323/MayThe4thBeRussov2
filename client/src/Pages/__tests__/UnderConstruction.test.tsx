import * as React from 'react'
import { shallow } from 'enzyme'
import UnderConstruction from '../UnderConstruction'

describe('UnderConstruction Page', () => {
  let wrapper: any
  beforeEach(() => (wrapper = shallow(<UnderConstruction />)))

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
