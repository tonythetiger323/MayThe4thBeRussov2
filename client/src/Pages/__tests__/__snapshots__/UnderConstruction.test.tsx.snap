// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnderConstruction Page should render correctly 1`] = `
<WithStyles(Typography)
  variant="h1"
>
  Under Construction
</WithStyles(Typography)>
`;
